system_hooks/src/addresses_constants.rs

```diff
@@ -38,6 +38,11 @@ pub const NONCE_HOLDER_HOOK_ADDRESS_LOW: u16 = 0x8003;
 pub const NONCE_HOLDER_HOOK_ADDRESS: B160 =
     B160::from_limbs([NONCE_HOLDER_HOOK_ADDRESS_LOW as u64, 0, 0]);
 
+// l2 interop root storage system hook (contract) needed for all envs (add interop root)
```

Comment:
nit: maybe makes sense to move this constant to bootloader to avoid confusion(since it's not a system hook)